ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/SearchParty.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.ccd.sdk.type;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.ccd.sdk.api.ComplexType;
+
+
+@NoArgsConstructor
+@Builder
+@Data
+@ComplexType(name = "Search Party", generate = false)
+public class SearchParty {
+  @JsonProperty("Name")
+  private String partyName;
```

Comment:
why prefix everything with party? it would be more concise with just name

Reply:
I think other teams have the same names and that is why in 
jira it has to be the same names